app/selenium_ui/bamboo/modules.py

```diff
@@ -31,8 +34,99 @@ def sub_measure():
     measure()
 
 
+def view_all_projects(webdriver, datasets):
+    @print_timing("selenium_view_all_projects")
+    def measure():
+        project_page = ProjectList(webdriver)
+        project_page.click_projects_button()
+        project_page.view_all_projects()
```

Comment:
`project_page.view_all_projects()` this is the useless method (and all methods like this in PR).
We open all projects page after we click on the Projects button. So, on 42 should be like `project_page.wait_for_page_loaded()`.
Also, test is called `view_all_projects`. So, maybe on 40 `projects_page` would be better, because its not ONE project page?